lib/utils/npm-helpers.js

```diff
@@ -0,0 +1,95 @@
+var path   = require('path');
+var fs     = require('fs-extra');
+var RSVP   = require('rsvp');
+var run    = require('./run');
+var rimraf = RSVP.denodeify(require('rimraf'));
+
+module.exports = {
+  
+  /**
+   * For a given scenario, get a list of NPM libraries that should be removed from
+   *  the node_modules folder, in order to effectively set the secenario up.
+   * 
+   * @param  {Object} scenario - a complete scenario object
+   * @return {Array} an array of strings -- names of folders in node_modules that
+   *    should be deleted before the scenario is "installed"
+   */
+  _npmFoldersToClearForScenario: function (scenario) {
+    return ['dependencies', 'devDependencies'].reduce(function (acc, depType) {
+      var deps = (scenario.npm && scenario.npm[depType]) ? scenario.npm[depType] : [];
+      return acc.concat(Object.keys(deps));
+    }, []);
+  },
+  
+  /**
+   * For a given scenarion, get a list of versioned npm packages that
+   *   should be installed.
+   * 
+   * @param  {Object} scenario - a complete scenario object
+   * @return {Array} an array of strings - packages with version
+   *    in npm notation (i.e., "rsvp@~1.0.2")
+   */
+  _npmFoldersToInstallForScenario: function (scenario) {
+    var packages = [];
+
+    ['dependencies', 'devDependencies'].forEach(function (depType) {
+      for (pkgName in (scenario.npm[depType] || [])) {
+        if (scenario.npm[depType][pkgName]) {
+          packages.push(pkgName + "@" + scenario.npm[depType][pkgName]);
+        }
+      }
+    });
+    debugger;
```

Comment:
yikes, sorry about that. I've now added `jshint` as part of `npm test` in addition to fixing this

https://github.com/kategengler/ember-try/commit/17ecc5966fda9de4e367a5a5c27b71a77c44fb24#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R15